### PR TITLE
Update README and config files with Moirai-1.1-R and fix Moirai-1.0-R model weights

### PR DESCRIPTION
**Pull request by gorold**
Made PRs on huggingface to fix the regression of Moirai-1.0-R model weights with the recent code changes
https://huggingface.co/Salesforce/moirai-1.0-R-small/discussions/7
https://huggingface.co/Salesforce/moirai-1.0-R-base/discussions/6
https://huggingface.co/Salesforce/moirai-1.0-R-large/discussions/6

Please help to test, I only managed to run some small tests on my local :)

The changes to model weights were just:
```
mm = MoiraiModule.from_pretrained("Salesforce/moirai-1.0-R-small")
mm.param_proj.proj.weights_logits.weight[:] = torch.roll(mm.param_proj.proj.weights_logits.weight, 2, dims=0)
mm.param_proj.proj.weights_logits.bias[:] = torch.roll(mm.param_proj.proj.weights_logits.bias, 2, dims=0)
mm.push_to_hub("Salesforce/moirai-1.0-R-small")
```

### CODE REVIEW

**Comment by liu-jc** [comment]
Hi @gorold,

Sorry, I don't quite understand. How these changes with torch.roll can help to fix the weights? We still have / dim in our codebase? https://github.com/SalesforceAIResearch/uni2ts/blob/27616d912b632fc77731b207e0b74970bed5870b/src/uni2ts/module/ts_embed.py#L178

**Comment by gorold** [comment]
1. The only weights affected are those where `self.dim > 1`, which in this case is just `param_proj.proj.weights_logits`.
2. What happened during training of Moirai 1.0 is that we had `weights_logits.out_features_ls = (32, 64, 128, 256, 512)`.
3. This meant that `weights_logits.weight[0:3]` contain the correct weights for patch sizes 32, 64, 128, where as the `weights_logits.weight[3:5]` are useless.
4. So we use `torch.roll` to push the weights into the correct positions for patch sizes 32, 64, 128, and leave 256, 512 in the first two positions as dummies.

**Review by liu-jc — COMMENTED** [review]
Now I got it. Do you think we should test all models and see if we can get the similar results on all datasets?

**Comment by gorold** [comment]
Ran it for small on the PF benchmark ex weather:

|index|dataset|test\_metrics/MSE\[mean\]|test\_metrics/MSE\[0\.5\]|test\_metrics/MAE\[0\.5\]|test\_metrics/MASE\[0\.5\]|test\_metrics/MAPE\[0\.5\]|test\_metrics/sMAPE\[0\.5\]|test\_metrics/MSIS|test\_metrics/RMSE\[mean\]|test\_metrics/NRMSE\[mean\]|test\_metrics/ND\[0\.5\]|test\_metrics/mean\_weighted\_sum\_quantile\_loss|
|---|---|---|---|---|---|---|---|---|---|---|---|---|
|0|electricity|3846122\.0|4047497\.75|219\.01437377929688|0\.9806302189826965|0\.1304870843887329|0\.13351434469223022|8\.014351844787598|1961\.1531982421875|0\.8221926093101501|0\.09181945025920868|0\.07224071025848389|
|1|solar-energy|1229\.113525390625|1439\.4451904296875|19\.23911476135254|1\.4672776460647583|2\.4206011295318604|1\.444728970527649|8\.45895767211914|35\.0587158203125|1\.138300895690918|0\.6246635317802429|0\.4719245433807373|
|2|walmart|27342336\.0|19882204\.0|2114\.254150390625|0\.9929503202438354|0\.24787983298301697|0\.17269687354564667|8\.747377395629883|5228\.990234375|0\.29910993576049805|0\.12094007432460785|0\.09698692709207535|
|3|istanbul\_traffic|135\.40859985351562|154\.15487670898438|8\.99282455444336|1\.0580122470855713|0\.5734052658081055|0\.3754161596298218|5\.677515506744385|11\.636520385742188|0\.31070834398269653|0\.24011866748332977|0\.17352095246315002|
|4|turkey\_power|755125\.1875|758453\.9375|358\.3751525878906|0\.9450287818908691|0\.5046748518943787|0\.3893551230430603|7\.002157211303711|868\.9793701171875|0\.14972686767578125|0\.061748750507831573|0\.048235006630420685|

Feel free to test on other sizes and datasets. You can directly download the model in the PR branch with the `revision` argument:
```
mm = MoiraiModule.from_pretrained("Salesforce/moirai-1.0-R-small", revision="pr/7")
```

Note that the PR number might be different for the other model sizes.

**Comment by gorold** [comment]
Base model:
|index|dataset|test\_metrics/MSE\[mean\]|test\_metrics/MSE\[0\.5\]|test\_metrics/MAE\[0\.5\]|test\_metrics/MASE\[0\.5\]|test\_metrics/MAPE\[0\.5\]|test\_metrics/sMAPE\[0\.5\]|test\_metrics/MSIS|test\_metrics/RMSE\[mean\]|test\_metrics/NRMSE\[mean\]|test\_metrics/ND\[0\.5\]|test\_metrics/mean\_weighted\_sum\_quantile\_loss|
|---|---|---|---|---|---|---|---|---|---|---|---|---|
|0|electricity|1708712\.5|1711201\.625|164\.1307373046875|0\.7915405631065369|0\.10031454265117645|0\.11076250672340393|6\.184873580932617|1307\.17724609375|0\.5480201840400696|0\.06881006807088852|0\.054687876254320145|
|1|solar-energy|1011\.0943603515625|1108\.408935546875|16\.981399536132812|1\.2911229133605957|2\.296311855316162|1\.4095485210418701|7\.017038345336914|31\.797710418701172|1\.0324212312698364|0\.5513591766357422|0\.41874560713768005|
|2|walmart|26299296\.0|19072352\.0|2049\.69384765625|0\.9657745957374573|0\.23114198446273804|0\.1677016019821167|8\.415294647216797|5128\.28369140625|0\.29334932565689087|0\.117247074842453|0\.09353204816579819|
|3|istanbul\_traffic|37\.16828918457031|40\.77793884277344|4\.562923431396484|0\.5369675755500793|0\.2586391270160675|0\.2553446292877197|3\.8279149532318115|6\.0965800285339355|0\.1627856343984604|0\.12183525413274765|0\.09833786636590958|
|4|turkey\_power|473797\.125|474377\.5625|295\.6066589355469|0\.8949130177497864|0\.16863861680030823|0\.37849825620651245|6\.532022476196289|688\.3292236328125|0\.11860048770904541|0\.050933610647916794|0\.040024567395448685|

**Review by liu-jc — APPROVED** [review]
LGTM
